src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/NoValueInRecordArrays/Index_NoValue.txt

```diff
@@ -0,0 +1,4 @@
+﻿// Feature NoValueInRecordArrays set
+
+>> Index([ { a: 10, b: "asdf"}, { a: 20, b: "second"}, { a: 30, b: "third"}], 1.1).b
```

Comment:
>1.1 [](http://example.com/codeflow?start=78&length=3)

nit: why 1.1?   We can have a dedicated Index() test for testing non-integer values to this arg. 

Or even to keep test simple, can we just do First()

Reply:
We probably don't need separate tests for Index and First. 
Since this is a table change, it will impact all table functions the same way.

Reply:
1.1 --> typo
Index & First - >those are only examples that illustrate the change but I agree all Table calls are potentially impacted